frontend/src/pages/ProjectDetails.tsx

```diff
@@ -41,36 +62,256 @@ const ProjectDetailsPage = () => {
         message="Sorry, the project you're looking for doesn't exist"
       />
     )
-  const params = ['idx_updated_at', 'idx_forks_count', 'idx_stars_count', 'idx_contributors_count']
-  const filteredIcons = getFilteredIcons(project, params)
-
-  const handleButtonClick = () => {
-    window.open(`/projects/contribute?q=${project.idx_name}`)
-  }
 
-  const SubmitButton = {
-    label: 'Contribute',
-    icon: <FontAwesomeIconWrapper icon="fa-solid fa-code-fork" />,
-    onclick: handleButtonClick,
-  }
+  const toggleContributors = () => setShowAllContributors(!showAllContributors)
+  const toggleLanguages = () => setShowAllLanguages(!showAllLanguages)
+  const toggleTopics = () => setShowAllTopics(!showAllTopics)
 
   return (
-    <div className="container mx-auto pb-16 pt-24 xl:max-w-full">
-      <div className="flex justify-center">
-        <Card
-          key={project.objectID}
-          title={project.idx_name}
-          url={project.idx_url}
-          summary={project.idx_summary}
-          level={level[`${project.idx_level as keyof typeof level}`]}
-          icons={filteredIcons}
-          leaders={project.idx_leaders}
-          topContributors={project.idx_top_contributors}
-          topics={project.idx_topics}
-          button={SubmitButton}
-        />
+    <div className="mt-20 min-h-screen bg-white p-8 text-gray-900 dark:bg-[#212529] dark:text-white">
```

Comment:
Can we change text color to match with our other pages? I think for dark theme it's `text-gray-300` and for light theme it's `text-grey-600`.